Sources/NetShears/UI/BodyDetailViewController.swift

```diff
@@ -19,37 +20,79 @@ final class BodyDetailViewController: UIViewController, ShowLoaderProtocol {
     static let kPadding: CGFloat = 10.0
     
     var bodyExportType: BodyExportType = .default
-
+    var textColor: UIColor {
+        if #available(iOS 12.0, *) {
+            switch traitCollection.userInterfaceStyle {
```

Comment:
```suggestion
            return UIColor { trait in
```
We can use this UIColor's dynamic provider init instead of the switch. It generates correct color even with changes of trait collection.

Reply:
iOS 13.0 or newer is only availalbe for iOS 13.0 or newer

Reply:
I guess there is no problem with the earlier versions as dark mode has been introduced on iOS 13

Reply:
done